pkgs/by-name/gi/gImageReader/package.nix

```diff
@@ -28,21 +30,24 @@
   gtkspell3,
   gtkspellmm,
   cairomm,
+  withQt6 ? false ? false,
+  kdePackages ? null,
+  qt6Packages ? null,
```

Comment:
I am not well versed into the kde ecosystem. Could this build for qt5 and qt6 or just qt6 (the latter is fine I think?) ? would it make sense to use `qtPackages` instead and in the all-packages.nix use `qtPackages = qt6Packages` like is usually done for versioned package sets